data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -28,42 +25,39 @@
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.servlet.mvc.method.annotation.StreamingResponseBody;
 
-import uk.gov.gchq.palisade.exception.ForbiddenException;
-import uk.gov.gchq.palisade.reader.request.DataReaderRequest;
-import uk.gov.gchq.palisade.service.data.config.StdSerialiserConfiguration;
-import uk.gov.gchq.palisade.service.data.config.StdSerialiserPrepopulationFactory;
-import uk.gov.gchq.palisade.service.data.model.AuditSuccessMessage;
+import uk.gov.gchq.palisade.service.data.model.AuditErrorMessage;
+import uk.gov.gchq.palisade.service.data.model.AuditableAuthorisedDataRequest;
+import uk.gov.gchq.palisade.service.data.model.AuditableDataResponse;
 import uk.gov.gchq.palisade.service.data.model.DataRequest;
-import uk.gov.gchq.palisade.service.data.service.AuditService;
-import uk.gov.gchq.palisade.service.data.service.DataService;
+import uk.gov.gchq.palisade.service.data.model.TokenMessagePair;
+import uk.gov.gchq.palisade.service.data.service.AuditMessageService;
+import uk.gov.gchq.palisade.service.data.service.AuditableDataService;
 
 import java.io.OutputStream;
-import java.util.concurrent.atomic.AtomicLong;
 
+/**
+ * Controller for data-service.  Provides the front end RESTFul web service for resources that have already been
+ * registered with the Palisade Service.  The request is in the form of information that will uniquely identify the
+ * resource request and will return a data stream of the response data.
+ */
 @RestController
-@RequestMapping(path = "/")
+@RequestMapping(path = "/api")
```

Comment:
Had been changed from an empyt string to "api".  I have checked with dbz120896 the correct answer is "data".  I've followed though on this and made the changes in this service to be inline with this convention.

Reply:
That 'data' probably comes from Traefik rather than from the service's rest controller (/cluster/palisade -> palisade-service, /cluster/resource -> filtered-resource-service, /cluster/data -> data-service, then we'd have /cluster/data/read/chunked if this string is blank, /cluster/data/data/read/chunked otherwise). Best left blank for now until we need it to be something different.

Reply:
Have done and rolled it back to a blank